src/main/resources/db/migrations/V001__CreateSchemaTables.sql

```diff
@@ -0,0 +1,171 @@
+    date DATE NOT NULL,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+
+CREATE TABLE public.booking_participants (
+    id UUID PRIMARY KEY NOT NULL,
+    participant_id UUID REFERENCES participants(id) NOT NULL,
+    booking_id UUID REFERENCES bookings(id) NOT NULL
+);
+
+CREATE TABLE public.users (
+    id UUID PRIMARY KEY NOT NULL,
+    first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL,
+    email VARCHAR(100) NOT NULL,
+    organisation VARCHAR(250),
+    phone VARCHAR(50),
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+CREATE TABLE public.roles (
+    id UUID PRIMARY KEY NOT NULL,
+    name VARCHAR(45) NOT NULL
+);
+
+CREATE TABLE public.app_access (
+    id UUID PRIMARY KEY NOT NULL,
+    user_id UUID REFERENCES users(id) NOT NULL,
+    court_id UUID REFERENCES courts(id) NOT NULL,
+    role_id UUID REFERENCES roles(id) NOT NULL,
+    last_access TIMESTAMPTZ,
+    active BOOLEAN DEFAULT TRUE,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+
+CREATE TABLE public.capture_sessions (
+    id UUID PRIMARY KEY NOT NULL,
+    booking_id UUID REFERENCES bookings(id) NOT NULL,
+    origin RECORDING_ORIGIN NOT NULL,
+    ingest_address VARCHAR(255),
+    live_output_url VARCHAR(100),
+    started_on TIMESTAMPTZ,
+    started_by_user_id UUID REFERENCES users(id),
+    finished_on TIMESTAMPTZ,
+    finished_by_user_id UUID REFERENCES users(id),
+    status RECORDING_STATUS,
+    deleted BOOLEAN DEFAULT FALSE
+);
+
+CREATE TABLE public.recording_versions (
+    id UUID PRIMARY KEY NOT NULL,
+    capture_session_id UUID REFERENCES capture_sessions(id) NOT NULL,
+    version INT NOT NULL,
+    url VARCHAR(255) NOT NULL,
+    filename VARCHAR(255) NOT NULL,
+    created_on TIMESTAMPTZ NOT NULL,
+    duration TIME,
+    edit_instruction JSON,
+    deleted BOOLEAN DEFAULT FALSE
+);
+
+CREATE TABLE public.share_recordings (
+    id UUID PRIMARY KEY NOT NULL,
+    capture_session_id UUID REFERENCES capture_sessions(id) NOT NULL,
+    shared_with_user_id UUID REFERENCES users(id) NOT NULL,
+    shared_by_user_id UUID REFERENCES users(id) NOT NULL,
+    created_on TIMESTAMPTZ DEFAULT NOW(),
+    deleted BOOLEAN DEFAULT FALSE
+);
+
+CREATE TABLE public.portal_access (
+    id UUID PRIMARY KEY NOT NULL,
+    user_id UUID REFERENCES users(id) NOT NULL,
+    password VARCHAR(45) NOT NULL,
+    last_access TIMESTAMPTZ,
+    invitation_sent BOOLEAN DEFAULT FALSE,
+    invitation_datetime TIMESTAMPTZ,
+    registered BOOLEAN DEFAULT FALSE,
+    registered_datetime TIMESTAMPTZ,
+    active BOOLEAN,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+CREATE TABLE public.audits (
```

Comment:
how are we going to handle auditing? i see we have an audit table and then some audit timestamps on a few models. to keep what we have here i suggest:

- going through and updating any timestamp column to us `at` instead of `on`
    - `created_at` for example
- each table have 3 timestamp columns?
    - `created_at`
    - `updated_at`
    - `deleted_at`
- we use a `deleted_at` timestamp column instead of a boolean. if the field is null for a recod then it has not been deleted.
- updating the column names in the audit table:
    - `id`
    - `auditable_type` -> essentially the table name
    - `auditable_id` -> id of the modified record
    - `user_id` -> user that performed the action
    - `source` -> don't think this should be an enum
    - `action` -> create/read/update/delete
    - `timestamp` -> time the action was performed
    - `payload` -> json object with extra details

Reply:
agree with suggestion to update timestamp columns to use consistent naming and also with the idea of using a deleted_at timestamp column instead of a boolean and changes to column names :D

With regards the below, can i get a bit more clarity pls:
- With regards the 'source' not being an enum, I think knowing the source of changes (e.g. from portal / app / etc) adds some valuble context to the audit logs. What do you suggest this change to? 
- With regards the 'actions' shouldn't we also include some specific based actions, like account lock or password reset? 
Let me know and other than that, ready to push in changes - thanks Lawrence

Reply:
I've raised this with Indy/Lawrie - if we think these fields might change in the future, VARCHAR would be more appropriate. I don't think they forsee the need for a change, but I'll keep you posted on this

Reply:
I think audits should be able to log any arbitrary data we see fit, so these fields should not be tied to the schema. I don't think it matter too much for now though until we implement auditing on the API

Reply:
We will revisit in the future